server/templates/obs.html

```diff
@@ -112,4 +112,18 @@ <h4>Products</h4>
         </div>
     {% endfor %}
     {% include 'pagination.html' %}
+
+    {% if owner %}
+    <div>
+    <fieldset class="admin">
+        <legend class="admin">Admin
+        Panel</legend>
+        The following actions are available, because you're the owner of the station:<br/>
+        <form method="post" action="/obs/delete/{{ obs.obs_id }}"><input type="button" class="danger" value="Delete Observation"
```

Comment:
Use ```<input type="submit" />```.
In ```onclick``` method you can return ```false``` if you want to cancel submitting.

Reply:
Updated as suggested.